tchannel/tchannel.py

```diff
@@ -232,6 +232,9 @@ def advertise(self, routers, name=None, timeout=None):
             The timeout (in seconds) for the initial advertise attempt.
             Defaults to 30 seconds.
 
+        :param router_file:
+            The host file that contains the routers information.
+
```

Comment:
Documentation should mention that exactly one of routers and router_file must be provided.